tests/foreman/api/test_host.py

```diff
@@ -66,6 +66,9 @@ def setUpClass(cls):
         cls.env = entities.Environment().search(
             query={'search': u'content_view="{0}"'.format(cls.cv.name)}
         )[0].read()
+        cls.env = entities.Environment(
```

Comment:
We have to found the right environment and add the location to it
```python
        cls.env = entities.Environment().search(
            query={'search': u'content_view="{0}" and organization_id={1}'.format(cls.cv.name, cls.org.id)}
        )[0].read()
        cls.env.location.append(cls.loc)
        cls.env.update(['location'])
```

Reply:
Thanks @ldjebran for your comment.
I also tried to do the same(adding location to environment).
what you suggested seem more correct to me as well.